brave/brave-core
# Use first name instead of full name in rewards uphold

Resolves https://github.com/brave/brave-browser/issues/9909

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- connect KYC account
- open panel and click on wallet verified
- make sure that you see `Hello, {firstName}!`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
